modules/programs/neomutt.nix

```diff
@@ -397,6 +402,12 @@ in {
           default = true;
         };
 
+      unmailboxes = mkOption {
```

Comment:
Would this name be easily understood by a neomutt user? I wonder if perhaps a more descriptive option name could be used?

Or at least expand the option description with a sentence on what the actual effect the option has.

Reply:
I will add more information to this option as I learned about this option only after scouring the mutt manual :+1: